### ISSUE TITLE
TSP Example doesn't work

### ISSUE THREAD

**Issue by Abraxos** [google/or-tools#170]
Hello,

I tried running the Python TSP example (or-tools/examples/python/tsp.py) so that I could try to modify it to fit my purposes, however, it doesn't seem to function properly and when run produces the following error:

```
Traceback (most recent call last):
  File "tsp.py", line 31, in <module>
    from ortools.constraint_solver import routing_enums_pb2
ImportError: cannot import name routing_enums_pb2
```

I was hoping someone could help me fix this (and subsequently fix it in the example)? Thank you!

**Comment by Abraxos** [comment]
I should note that I got to this point by downloading the python binaries on ubuntu 14.04 using the following approach:

```
$ python setup.py install --user
```

**Comment by Abraxos** [comment]
Even compiling from source and running as shown below does not fix the issue =(

```
eugene@wall-e:~/Desktop$ PYTHONPATH=~/Downloads/or-tools-2015-12/src python2.7 tsp.py
Traceback (most recent call last):
  File "tsp.py", line 31, in <module>
    from ortools.constraint_solver import routing_enums_pb2
ImportError: cannot import name routing_enums_pb2
```

**Comment by lperron** [comment]
What did you do to compile from source?
Did you get the latest version?
Did you run make third_party recently?

Thanks

**Comment by Abraxos** [comment]
I downloaded the zipped version of the source code on the github page and followed the instructions here: https://developers.google.com/optimization/installing#installing-from-source

and I ran `make third_party` right before running `make install_python_modules` as per the instructions

**Comment by hxd1011** [comment]
I had many problems on "ImportError", I think the problem is the privilege. I tried to install to user, but there are other problems.

my solution would be "sudo" everything...
When you install do 

sudo python setup.py install

when use use it, do

sudo python your_examples.py

**Comment by lperron** [comment]
The zip version is out of date if it is linked to a release.
You need a fresh git clone.

I know there are problems with debian and setup.py --user. There are some workaround (complex) on the net.

Thanks

**Comment by Abraxos** [comment]
Gotcha, well I got it to compile from source on Debian and that got the tsp.py example working so I guess I am good to go. I will publish my build instructions to my website (if you do not mind) in case someone comes up on the same issue. Thank you for your help.

Notably I did get it to compile and run without sudo, only needing sudo in order to install the package to the system on the last step. No offense, but needing running normal programs as root makes me very uncomfortable so I am glad I found a way around that.
